### PR TITLE
Support `INSERT IGNORE` in `MySql` and `GenericDialect`

### PR DESCRIPTION
**Pull request by emin100**
SQL: INSERT IGNORE INTO tasks (title, priority) VALUES ('Test Some Inserts', 1)
ERROR:  sql parser error: Expected SELECT, VALUES, or a subquery in the query body, found: INTO at Line: 1, Column 15

### CODE REVIEW

File: src/parser/mod.rs [hunk 1]
```diff
@@ -6697,6 +6697,8 @@ impl<'a> Parser<'a> {
             None
         };
 
+        let ignore = dialect_of!(self is MySqlDialect) && self.parse_keyword(Keyword::IGNORE);
```
Comment (alamb) [inline src/parser/mod.rs:6700]: Can you please also add support for GenericDialect?

**Review by alamb — COMMENTED** [review]
Thank you for the contribution @emin100 -- this PR fails a CI check

I made a PR with a proposed fix that targets this branch -- please give it a look: https://github.com/emin100/sqlparser-rs/pull/1

File: src/parser/mod.rs (hunk 1 above)
Comment (emin100) [reply src/parser/mod.rs]: Generic support added

**Review by alamb — APPROVED** [review]
Thank you @emin100

**Pull request merged**